Framework/Core/include/Framework/runDataProcessing.h

```diff
@@ -128,6 +128,13 @@ int main(int argc, char** argv)
     workflowOptions.push_back(ConfigParamSpec{"pipeline", VariantType::String, "", {"override default pipeline size"}});
     workflowOptions.push_back(ConfigParamSpec{"dangling-outputs-policy", VariantType::String, "file", {"what to do with dangling outputs. file: write to file, fairmq: send to output proxy"}});
 
+    // options for AOD writer
+    workflowOptions.push_back(ConfigParamSpec{"aod-writer-json", VariantType::String, {"Name of the json configuration file"}});
+    workflowOptions.push_back(ConfigParamSpec{"res-file", VariantType::String, {"Default name of the output file"}});
```

Comment:
We could also prepend aod-writer to the other 3 options

Reply:
i agree.

Reply:
and then also aod-reader-...for all the reader options?

Reply:
We only have one reader, so I would not touch that for now.